src/py/flwr/server/server_test.py

```diff
@@ -182,3 +199,82 @@ def test_set_max_workers() -> None:
 
     # Assert
     assert server.max_workers == 42
+
+
+def test_setup_client_auth() -> None:
+    """Test setup client authentication."""
+    # Generate keys
+    _, first_public_key = generate_key_pairs()
+    server_public_key = (
+        b"ecdsa-sha2-nistp384 "
+        b"AAAAE2VjZHNhLXNoYTItbmlzdHAzODQAAAAIbmlzdHAzODQAAABhBIqtP/EvrgBYukcjRJT9zVLXE"
+        b"fykvVvT/QcHXuxCNu83SyCwedk3nNZxy5rZ1f8KoU+OSGmum5I9BxnWcLeBC+TGqpifTUSNwa/riV"
+        b"oJGcN/SxF3euqQg58YePORhos/Ug=="
+    )
+    server_private_key = b"""-----BEGIN OPENSSH PRIVATE KEY-----
+    b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAiAAAABNlY2RzYS
+    1zaGEyLW5pc3RwMzg0AAAACG5pc3RwMzg0AAAAYQSKrT/xL64AWLpHI0SU/c1S1xH8pL1b
+    0/0HB17sQjbvN0sgsHnZN5zWccua2dX/CqFPjkhprpuSPQcZ1nC3gQvkxqqYn01EjcGv64
+    laCRnDf0sRd3rqkIOfGHjzkYaLP1IAAADw/rbMO/62zDsAAAATZWNkc2Etc2hhMi1uaXN0
+    cDM4NAAAAAhuaXN0cDM4NAAAAGEEiq0/8S+uAFi6RyNElP3NUtcR/KS9W9P9Bwde7EI27z
+    dLILB52Tec1nHLmtnV/wqhT45Iaa6bkj0HGdZwt4EL5MaqmJ9NRI3Br+uJWgkZw39LEXd6
+    6pCDnxh485GGiz9SAAAAMQDQmvP7JeFNBDvo1VXciQF0Wv3/DCcj9x0kUABuX1gxb42Iw3
+    v7FOEco/enMaS4URwAAAAnZGFuaWVsbnVncmFoYUBEYW5pZWxzLU1hY0Jvb2stUHJvLmxv
+    Y2Fs
+    -----END OPENSSH PRIVATE KEY-----"""
```

Comment:
Can we generate these dynamically, instead of hardcoding them here?

https://stackoverflow.com/questions/2466401/how-to-generate-ssh-key-pairs-with-python

Reply:
Done.